wct-core/src/main/resources/defaultH3Profile.cxml

```diff
@@ -134,48 +134,6 @@ metadata.description=Basic crawl starting with useful defaults
           <property name="regexList">
            <list>
                <value>.*rss.*</value>
-               <value>.*Rss.*</value>
```

Comment:
Was this regex list too specific to natlib crawls?

Reply:
The rss one no, but there were a few others. I've added back in a few generalised ones, and fixed the regexs to more tightly match.